lib/index.js

```diff
@@ -259,7 +263,12 @@ exports.tooManyRequests = function (message, data) {
 
 exports.internal = function (message, data, statusCode) {
 
-    var error = (data instanceof Error ? exports.wrap(data, statusCode, message) : exports.create(statusCode || 500, message));
+    return internals.internal(message, data, statusCode, exports.internal);
+};
+
+internals.internal = function (message, data, statusCode, ctor) {
```

Comment:
It's just not very descriptive of what it does. If someone wants to add another status code based boom type, they'll have to go looking to see what `internals.internal` does.